### ISSUE TITLE
easybib.com: ads

### ISSUE THREAD

**Issue by subby2006** [uBlockOrigin/uAssets#21025]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.easybib.com/mla-format/website-citation/new`

### Category

Forced ads when clicking 'complete citation'

### Description

Webpages forces you to view an ad or interact with a TrueX/Infillion survey before letting you complete citation
Unfortunately I forgot to take a screenshot of the ad and now it doesn't show up anymore, even with using private browsing + Cloudflare WARP 

### Other extensions used

Greasemonkey, LocalCDN, Bypass Paywalls Clean

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.54.0
Firefox: 119
filterset (summary):
 network: 293124
 cosmetic: 85131
 scriptlet: 40074
 html: 1326
listset (total-discarded, last-updated):
 added:
  https://malware-filter.gitlab.io/pup-filter/pup-filter.txt: 189-0, 47m
  https://raw.githubusercontent.com/JourneyOver/adfilt/master/filters/RickrollLinkIdentifier.txt: 9-0, never
  https://raw.githubusercontent.com/Spam404/lists/master/adblock-list.txt: 7253-1, 47m
  https://winhelp2002.mvps.org/hosts.txt: 8729-329, now
  adguard-generic: 80387-7321, 47m
  ublock-annoyances: 5920-11, now
  dpollock-0: 11586-1644, now
  adguard-spyware-url: 1342-92, 47m
  curben-phishing: 88422-27, 47m
  adguard-spyware: 74939-1028, 47m
 default:
  user-filters: 129-0, never
  ublock-filters: 36769-262, now
  ublock-badware: 7537-16, now
  ublock-privacy: 931-4, now
  ublock-unbreak: 2203-1, now
  easylist: 75620-834, now
  easyprivacy: 33037-17121, now
  urlhaus-1: 9695-0, 47m
  plowe-0: 3779-1, now
  ublock-quick-fixes: 268-68, 47m
filterset (user): [array of 132 redacted]
trustedset:
 added: [array of 20 redacted]
 removed:
  about-scheme
switchRuleset:
 added: [array of 2 redacted]
 removed:
  no-csp-reports: * true
userSettings:
 advancedUserEnabled: true
 cloudStorageEnabled: true
 webrtcIPAddressHidden: true
hiddenSettings:
 filterAuthorMode: true
supportStats:
 allReadyAfter: 821 ms (selfie)
 maxAssetCacheWait: 488 ms
popupPanel:
 blocked: 619
 network:
  33across.com: 51
  3lift.com: 16
  adnxs.com: 16
  adsrvr.org: 1
  bidswitch.net: 16
  casalemedia.com: 15
  chegg.com: 327
  confiant-integrations.net: 1
  criteo.com: 15
  criteo.net: 1
  doubleclick.net: 1
  google.com: 5
  googlesyndication.com: 1
  googletagmanager.com: 1
  gumgum.com: 15
  id5-sync.com: 1
  imasdk.googleapis.com: 1
  indexww.com: 1
  onetrust.com: 2
  openx.net: 16
  optimizely.com: 4
  permutive.app: 1
  pubmatic.com: 16
  quantserve.com: 1
  rightfulfall.com: 2
  rubiconproject.com: 46
  scorecardresearch.com: 1
  yahoo.com: 45
 extended:
  ##.adds-wrapper
  ##div[class^="styled__SideView"]
  ##+js(abort-on-property-read, SBMGlobal.run.gramCallback)
  ##+js(addEventListener-defuser, load, (!i))
```
</details>

**Comment by mapx-** [comment]
You should provide a complete set of steps to reproduce whatever issue (required account ?, exactly what to write in the various fields etc)

**Comment by Omega-6** [comment]
> You should provide a complete set of steps to reproduce whatever issue (required account ?, exactly what to write in the various fields etc)

I reproduced, just proceed as if you were a student trying to cite a paper / webpage, and then it asks as "payment" either a subscription or viewing a video

**Comment by huyszn** [comment]
I was also able to reproduce it (no account required), but I had to do a second citation.

The video did not load for me, so I couldn't complete the citation. https://github.com/uBlockOrigin/uAssets/assets/46979193/4442cc8a-13bc-4672-a73a-c345a702866e

Logger output when I select the sponsored message option: https://github.com/uBlockOrigin/uAssets/assets/46979193/46e58af3-77f9-4dfe-a8d8-d913543d99df

I used the exception `media.truex.com#@##ad_container` to make the sponsored video load, but I had to wait 30 seconds to be able to complete the citation.
https://github.com/uBlockOrigin/uAssets/assets/46979193/c57577cb-e4ff-4000-92d7-8c365cb5722b
